src/main/kotlin/com/miquido/stringstranslator/parsing/spreadsheet/StringHtmlAwareEscape.kt

```diff
@@ -0,0 +1,23 @@
+package com.miquido.stringstranslator.parsing.spreadsheet
+
+import com.miquido.stringstranslator.extensions.escape
+
+class StringHtmlAwareEscape(
```

Comment:
Optymalizacja pamięci (i poprawa architektury?) :
* zamienić klasę na coś w stylu `StringHtmlAwareEscaper` i przekazywać 'text' jako parametr do 'value'.
* escapeMap mogłoby zostać jako property.
* StringHtmlAwareEscaper mógłby być injectowany (ew. tworzony raz dla danego Writera)

Aktualnie chyba dość dużo jest tworzonych obiektów 'StringHtmlAwareEscape' (są tworzone w pętlach) i mają bardzo krótki czas życia, przez co GC ma dużo pracy :)

Nie wiem czy może to wpłynąć istotnie na czas działania, ale wydaje mi się, że architektura też byłaby dzięki temu trochę lepsza.